modules/core/04-channel/keeper/timeout.go

```diff
@@ -68,10 +68,9 @@ func (k Keeper) TimeoutPacket(
 		return err
 	}
 
-	timeoutHeight := packet.GetTimeoutHeight()
-	if (timeoutHeight.IsZero() || proofHeight.LT(timeoutHeight)) &&
-		(packet.GetTimeoutTimestamp() == 0 || proofTimestamp < packet.GetTimeoutTimestamp()) {
-		return errorsmod.Wrap(types.ErrPacketTimeout, "packet timeout has not been reached for height or timestamp")
```

Comment:
tangential: `types.ErrPacketTimeout` is solely used further down in `timeout.go` atm (see https://github.com/cosmos/ibc-go/blob/6c6ce879fc0201af907b864f955ce509222e12ab/modules/core/04-channel/keeper/timeout.go#L181).

Considering we've changed timeouts here and in `packet.go` to refer to `ErrTimeoutElapsed`, does it make sense to replace that one too (we now have unified timeout handling for both packets + channel upgrades)

Reply:
good question. I'm gonna leave as is. I don't have access to the proof height/timestamp in TimeoutExecuted and given that this is the error string in the upgrade, it likely should say packet timeout. I could do something like `errorsmod.Wrap(types.ErrTimeoutElapsed, "packet timed out")`. That or leaving as `types.ErrPacketTimeout` is fine by me

Reply:
on second thought, I added the wrapped error

Reply:
beautiful!

Reply:
I'll open issue re: deprecation later if no one beats me to it else this easily ends up forgotten